### ISSUE TITLE
examples\corelib\helloworld example is broken

### ISSUE THREAD

**Issue by saulery** [graemeg/fpGUI#70]
The example in examples\corelib\helloworld doesn't compile.

**Comment by graemeg** [comment]
This issue has been resolved in commit 64bd8ac. Since the "alien_window" changes, this demo app was
not updated to reflect the new internal design of fpGUI.

**Comment by mhekeler** [comment]
I think another problem for a new user is that the file `extrafpc.cfg` with compiler switches is missing in this directory.
The `compileall.sh` script requires this file for compiling `helloworld.lpr`
This can be solved by copying extrafpc.cfg from any example directory (no matter from which one)

```
cp ../menutest/extrafpc.cfg .
fpc @extrafpc.cfg helloworld.lpr

```
